src/layout/Grid.js

```diff
@@ -87,7 +87,9 @@
         return this;
     };
 
-    Grid.prototype.getCell = function (row, col) {
+    Grid.prototype.getCell = function (_row, _col) {
+        var row = _row * this.cellDensity();
+        var col = _col * this.cellDensity();
```

Comment:
Valid point, yes I should have considered getCell might be used outside of the layout/Grid file.

I suppose a new method **getDensityCell()** should be created/used inside the layout/Grid file and getCell should be as you have it in this PR.